app/controlplane/internal/biz/signing_test.go

```diff
@@ -0,0 +1,98 @@
+//
+// Copyright 2024 The Chainloop Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package biz_test
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/rand"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/pem"
+	"fmt"
+	"testing"
+
+	"github.com/chainloop-dev/chainloop/app/controlplane/internal/biz"
+	"github.com/sigstore/fulcio/pkg/ca/ephemeralca"
+	"github.com/sigstore/sigstore/pkg/cryptoutils"
+	"github.com/stretchr/testify/suite"
+)
+
+type signingUseCaseTestSuite struct {
+	suite.Suite
+	uc  *biz.SigningUseCase
+	csr []byte
+}
+
+func (s *signingUseCaseTestSuite) TestSigningUseCase_CreateSigningCert() {
```

Comment:
nitpick: I think this is smth I asked @javirln once, but is using `_` i.e `TestSigningUseCase_CreateSigningCert` go idiomatic?

Reply:
It's the suggestion from the IDE, I believe it's some "idiomatic", but I don't think there is a written rule.